src/Resources/config/services.yaml

```diff
@@ -0,0 +1,33 @@
+services:
```

Comment:
why yaml?

Reply:
why not? tired of xml. didn't find anywhere that it has to be done with xml no?

Reply:
https://symfony.com/doc/current/bundles/configuration.html#supporting-xml